frontend/src/pages/contactus/index.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
```

Comment:
No need to import React here. I'd recommend removing this because React is already imported in the base index file